doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,165 @@
+scikit-image project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+avoided.
+
+Roles And Responsibilities
+==========================
+
+Contributors
+------------
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the
+:ref:`contributors guide <howto_contribute>`.
+
+Core developers
+---------------
+Core developers are community members that have demonstrated continued
+commitment to the project through ongoing engagement with the community. They
+have shown they can be trusted to maintain scikit-image with care. Being a core
+developer allows contributors to more easily carry on with their project
+related activities by giving them direct access to the project’s repository and
+is represented as being an organization member on the scikit-image
+`GitHub organization <https://github.com/orgs/scikit-image/people>`_.
+Core developers are expected to review code contributions, can merge approved
+pull requests, can cast votes for and against merging a pull-request, and can
+be involved in deciding major changes to the API. Core developers are expected
+to adhere to the :ref:`core developer guide <core_dev>`.
+
+New core developers can be nominated by any existing core developer.
+Discussion about new core developer nominations is one of the few activities
+that takes place on the project's private management list. The decision to
+invite a new core developer must be made by “lazy consensus”, meaning unanimous
+agreement by all responding existing core developers. Invitation must take
+place at least one week after initial nomination, to allow existing members
+time to voice any objections.
+
+Steering Council
+----------------
+The Steering Council (SC) members are core developers who have additional
+responsibilities to ensure the smooth running of the project. SC members are
+expected to participate in strategic planning, approve changes to the
+governance model, and make decisions about funding granted to the project
+itself. (Funding to community members is theirs to pursue and
+manage.) The purpose of the SC is to ensure smooth progress from the
+big-picture perspective. Changes that impact the full project require a
+synthetic analysis and a consensus that is both explicit and informed. When the
+core developer community (including the SC members) fails to reach such a
+consensus in a reasonable timeframe, the SC is the entity to resolve the issue.
+
+Membership of the SC is by nomination by a core developer. A nomination will
```

Comment:
This may require a more structured approach.  Define the number of positions, fill them by perhaps having the core team nominate and vote.  And then do an annual check-in with existing SC members to see whether they want to serve again.

Reply:
maybe every two years? I don't relish a long discussion about all this every Thanksgiving. =P I'm also not sure that we want to *strictly* constrain the size of the SC, so there should be a mechanism for changing the size, too. Thoughts?